src/stores/authenticationStore.ts

```diff
@@ -20,6 +20,11 @@ export const useAuthenticationStore = defineStore('authentication', {
       return useRootStore().userKeycloakGuid
     },
 
+    /** The user's Full Name. */
+    getUserFullName (): string {
+      return useRootStore().userFullName
+    },
```

Comment:
Either move this to rootStore.ts or move the property (and setter) to this store.

My vote is to move this to the root store. This store uses `RootStateIF` (unfortunately named) which contains user account and auth data. Meanwhile the root store (which uses `StateIF`) contains more "info" type data.

Reply:
Sev, I moved it to the rootStore and did the changes accordingly.